lib/shortcuts/s3-kinesis-firehose.js

```diff
@@ -41,7 +53,24 @@ class S3KinesisFirehose extends KinesisFirehoseBase {
       DestinationBucket,
       Prefix = { 'Fn::Join': ['', ['raw/', LogicalName, '/']] },
       BufferingIntervalInSeconds = 900,
-      BufferingSizeInMBs = 128
+      BufferingSizeInMBs = 128,
+      AlarmName = { 'Fn::Sub': `\${AWS::StackName}-${LogicalName}-Freshness-\${AWS::Region}` },
+      AlarmDescription = {
+        'Fn::Sub': [
+          'Freshness alarm for ${AWS::StackName}-${name} kinesis firehose in ${AWS::StackName} stack',
+          { name: LogicalName }
+        ]
+      },
+      AlarmActions = [],
+      Period = 60,
+      EvaluationPeriods = 1,
+      Statistic = 'Average',
```

Comment:
It should be on Max. For each time interval there is one data point per firehose "shard". The case you really want to catch here is one where shards are imbalanced, and a one shard is significantly more latent than another.

Reply:
cool. done via 854201c